### DISCUSSION TITLE
Version 6 - It is safe to migrate and how to do this?

### DISCUSSION THREAD

**Discussion by 7iomka** [mantinedev/mantine#3370] [Q&A]
I see you have a lot of fixes rolled out at once in version 6, which has not yet been published.
It has a pretty decent list of breaking changes. Is there any way to publish this version currently (you can call it canary) so I can install it and start making edits? I use yarn and always do yarn up @mantine/*, but how to use version 6 which is apparently only on github for now I don't know.
And yes, what are your plans for a general release date for version 6?

**Comment by rtivital** [comment]
It is still too early to upgrade, not all breaking changes are implemented yet. Alpha version will be published in about a month. Release is planed in March-April.

**Comment by 7iomka** [comment]
In other words we won't receive any updates till the march-april?)

**Reply by rtivital** [reply]
Yes, it is correct
